bin/lowly.js

```diff
@@ -44,4 +48,46 @@ input.forEach(file => {
     }).catch(err => {
       console.error(err)
     })
+}
+
+input.forEach(ip => {
```

Comment:
You could use the module [file-extension](https://www.npmjs.com/package/file-extension) to see that.